tern/command_lib/base.yml

```diff
@@ -22,6 +23,8 @@ tdnf:
   path:
     - 'usr/bin/tdnf' # don't put forward slash here as os.path.join will think it is the root directory on the host
   shell: '/usr/bin/bash'
+  pkg_delimiter:
+    - '-'
```

Comment:
Is your expectation that there will be more than 1 package delimiter?

Reply:
Yes, for example in the case of `pip` you can use `>=` or `<=` or various combinations involving `~`.

Reply:
But now I am thinking that I may want to pass in a list to `get_pkg_name` instead of just a single string in the case of multiple delimiters.

Reply:
I wonder if we need to make the distinction between what delimiter we use for pinning to a version vs what delimiters get used in the Dockerfile. I would think that the pinning delimiter used would be a string but for parsing out the package name from the package name + version string that might be taken care of by a regex.

Reply:
For example: `RUN pip install -y tern>=1.0` in a Dockerfile will be transcribed as `RUN pip install -y tern==1.0.1`. So I guess `pinning_separator` for `pip` is `==` but the delimiters can be any combination of `>=~^`.

Reply:
Ah! Ok, I see, that makes sense. I think we can put `pinning_separator` and `pkg_delimiters` both in base.yml and have but have `pkg_delimiters` be list of potential values and `pinning_separator` just be the one string? 

If that is the case, `get_pkg_name` would still need to take a list of delimiters to account for in finding the package name. Somewhere else in the code will take care of swapping the `pkg_delimiter` with the `pinning_separator` for the frozen Dockerfile. Is that similar to what you're thinking as well?